test/tests/functional/pbs_user_reliability.py

```diff
@@ -111,7 +112,7 @@ def test_create_resv_from_job_using_qsub(self):
 
         now = time.time()
 
-        a = {ATTR_W: 'create_resv_from_job=1'}
+        a = {ATTR_W: 'create_resv_from_job=True'}
```

Comment:
Is this change needed? If not, I'll say just leave it. 
There is already a PR that Birbal raised to change the attribute value in hooks to use 'True' instead of 1 because it fails when a numeric value is given in the hook. There seems to be an issue in our hook implementation.
Ideally, the attribute should accept both '1' or 'True' (irrespective that it is being set from qsub, qalter or hooks), and having it set to 1 here proves that point.

Reply:
I changed it because I wanted the second part of the test I was adding to request False.  If I leave this as 1, then I'll have to request 0.  This is a PTL test, not a hook, so birbal's PR doesn't affect this.  PTL just wraps this up and sends it to qsub, and qsub correctly deals with decoding booleans.  If you want me to change this back to 1 (and request 0 further on), I will.